app/views/wpcc/your_results/index.html.erb

```diff
@@ -23,16 +23,26 @@
 
     <div class="results__row">
       <% @schedule.each do |period| %>
-        <div class="results__period results__period-<%= period.name %>">
+        <div class="results__period results__period-<%= period.name %>" data-dough-component="PopupTip">
           <h3 class="results__period-title"><%= period.title %></h3>
 
           <p class="results__period-heading"><%= t('wpcc.results.period.contribution_heading.yours') %></p>
           <p class="results__period-details results__period-employee-contribution">
             <%= period.employee_contribution %>
           </p>
-          <p class="results__period-parenthesis results__period-tax-relief">
-            <%= period.tax_relief %>
+          <p class="results__period-parenthesis">
+            <span class="results__period-tax-relief">
+              <%= period.tax_relief %>
+            </span>
+            <button type="button" class="popup-tip__button" data-dough-popup-trigger>
+              <span aria-hidden="true">i</span>
+              <span class="visually-hidden"><%= t('wpcc.tooltip_show') %></span>
+            </button>
```

Comment:
Ah, in fact I was confused with the close button on the tooltip. Good point, will do.